### DISCUSSION TITLE
Schema version should be required?

### DISCUSSION THREAD

**Discussion by jlamb0423** [CMSgov/price-transparency-guide#481] [Q&A] [answered]
The allow amounts and in network schemas suggests The version of the schema for the produced information is NOT required.  Yet, we have to specify the version when running the validator.  Shouldn't the schema version be required otherwise how will we know which version to run the validator against?

**Comment by shaselton-usds** [comment, marked as answer]
Yeah, that should be required.  PR here: https://github.com/CMSgov/price-transparency-guide/pull/488

**Reply by jlamb0423** [reply]
Moreover, shouldn't the file name have a suffix that includes the version number.  Something like "v1_1_0".  This way we can easily parse the version number to initiate the CMS validation check.

**Reply by shaselton-usds** [reply]
The version number is going to stay in the file -- parsing this in an automated way should be straightforward that wouldn't need to rely on the file name.
